features/step_definitions/evidence_hub/search_steps.rb

```diff
@@ -2,11 +2,24 @@
   evidence_summaries_page.load(locale: 'en')
 end
 
-When('I search on evidence hub the keyword {string}') do |keyword|
+Given("I search based on some filters") do
```

Comment:
I know we don't technically cover rubocop over features, but I think we should stick to the convention of single quotes on steps 
`Given('I search based on some filters') do`